README.md

```diff
@@ -736,6 +736,7 @@ We currently support the following hooks.
 * pre_commit - after validations or interactive wizard have completed, all parameter values are set but not yet stored in the answer file
 * pre  - just before puppet is executed to converge system, after parameter values are stored in the answer file
 * post  - just after puppet is executed to converge system
+* exit  - happens as part of exit handling
```

Comment:
I'd consider `pre_exit` to be consistent with `pre_migrations`, `pre_values` and `pre_validations`.